config/kron.py

```diff
@@ -0,0 +1,8 @@
+# Enable Kron mags
+# 'config' is a SourceMeasurementConfig
+
+try:
+    import lsst.meas.extensions.photometryKron
+    config.plugins.names |= ["ext_photometryKron_KronFlux"]
+except ImportError:
+    print("Cannot import lsst.meas.extensions.photometryKron: disabling Kron measurements")
```

Comment:
Can this be a more proper log message, or does that not work in config files?

Reply:
Changed to:
```python
except ImportError:
    import logging
    logging.getLogger("lsst.obs.decam.config").warning("Cannot import lsst.meas.extensions.photometryKron:"
                                                       " disabling Kron measurements")
```